### PR TITLE
Storage Feature Added

### PR DESCRIPTION
**Pull request by gabrielyea**
# Local storage
- Data is saved when input fields are modified.
- Data is saved as one object in local storage.
- Forms fill automatically if data is found.

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters error ✔️ 
- Your project is excellent and it works well ✔️ 

## Aspects to improve ♻️

- [x] It's important to use a `window.onload()` function to use the storage set/get item. I kindly invite you to add this to your project. [Check this](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload) if you want to read more about it.
- Example:
```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/main.js [hunk 1]
```diff
@@ -31,10 +32,14 @@ projectPopUp.exitBtn.addEventListener(('click'), () => {
   projectPopUp.closePopUp();
 });
 
-window.addEventListener('load', initProjects());
+window.addEventListener('load', () => {
+  initProjects();
+  loadInputData();
+});
```
Comment (gabrielyea) [inline scripts/main.js:38]: Hello! @vichuge
I do load the data on load using the 'load' event. It works the same as specified in the [official docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event)

Comment (vichuge) [reply scripts/main.js:38]: Yep, you're right, that was my bad. Just ask again for a CR and then the next code reviewer should accept your project.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**